addons/mail/static/src/new/discuss/channel_member_list.xml

```diff
@@ -30,7 +30,7 @@
             <div class="o-mail-background-inherit o-mail-channel-member-avatar-container position-relative d-flex ms-4 flex-shrink-0">
                 <img class="w-100 h-100 rounded-circle o_object_fit_cover o_redirect"
                     t-att-src="member.persona.avatarUrl"/>
-                <PartnerImStatus className="'position-absolute bottom-0 end-0'" partner="member"/>
+                <PartnerImStatus className="'position-absolute bottom-0 end-0'" partner="member.persona"/>
```

Comment:
Should we rename all partner to persona ? Feel weird to pass a persona model to the partner props

Reply:
will do that as a follow-up PR